packages/components/src/mobile/bottom-sheet/cycle-picker-cell.native.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { findIndex } from 'lodash';
+/**
+ * Internal dependencies
+ */
+import Cell from './cell';
+
+export default function BottomSheetCyclePickerCell( props ) {
+	const {
+		value,
+		options,
+		onChangeValue,
+		...cellProps
+	} = props;
+
+	const cycleOptionValue = () => {
```

Comment:
A better name for this function imo would be `nextOptionValue`. Cycle does indicate that the possible options form a cycle but it's hard to understand that this is returning the next one from the currently selected one.
I think readability could be improved here as well, something as simple as this would really help I think
```
const nextOptionValue = () => {
	const selectedOptionIndex = findIndex( options, [ 'value', value ] );
	return options[ ( selectedOptionIndex + 1 ) % options.length ].value;
};
```

Reply:
We might want to handle the case where options is en empty array as well ;)

Reply:
Updated constant name 9aa8aa7d5d5aad5592b40096f6ddb6e1d5c19138 . Added checks here 12ebc85ea833e3ebce644bffdf7273448c12151d  so the image block will hide image size options if sizes / default are ever empty or not matching, and a check so that the `CycleSelectControl`  `value` will just be blank if it is ever passed empty options.